stream_alert_cli/terraform_generate.py

```diff
@@ -303,6 +316,8 @@ def generate_kinesis(cluster_name, cluster_dict, config):
         's3_logging_bucket': logging_bucket
     }
 
+    return True
```

Comment:
@jacknagz  in a handful of these functions it looks like we return `True` and there's really no possibility of the function returning a falsey value. Am I overlooking something? If that's the case, there's not much benefit to having a check for the return value for these function calls.

That said, I do see a few functions below that can return `False`, so it definitely applies some places.